poetry.lock

```diff
@@ -1,9 +1,10 @@
-# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry and should not be changed by hand.
```

Comment:
Looks like this was done with an older version of poetry.

Reply:
reverted in e2d7f2b48a1a3454a33e89c1194e6df8b98e046f